### DISCUSSION TITLE
ESP32 Kullanarak Göcük Altlarinin Izlenmesi

### DISCUSSION THREAD

**Discussion by alihansencan** [acikkaynak/deprem-yardim-projesi#31] [Ideas]
Sevgili Kadir (@kadiryskky) arkadasimizin ESP32 ler kullanilarak göcük altlarinin izlenmesi hakkinda bir fikri var. Kendisinin paylastigi videolari da ekliyorum.

https://www.youtube.com/watch?v=OejZ2JsHZkc

https://www.youtube.com/watch?v=vPcJ5HOVSRQ

**Comment by Alone1206** [comment]
Fikrin detaylandırılması lazım. Yani bu nerede kullanılacak tam olarak? Eğer nodemcu'ya kamera takıp yıkılan binadan aşağı sallandıralım dersek kameradan gelen yayın birkaç kat betondan sonra mozaiklenmiş bir görüntüye dönecektir. Bu noktada eğer fikir böyleyse uzun kablolu yılan kameralar kablolu olmaları neticesinde görüntü kaybı olmadan daha efektif sonuç verebilir. Eğer tabi kullanım fikri başkaysa 
ona göre bakmak lazım. Her türlü IoT ve nodemcu programlanması ihtiyacınız için bana yazabilirsiniz.

**Reply by siniradam** [reply]
LoRa konusunda bilgin var mi? Discord'da beni bulabilirsin: tinman#7219

**Comment by ramazansubasi** [comment]
Detaylı bir konuşmak lazım. Teknik olarak yapılması artık çok kolay işler bunlar, sahada nasıl amaca hizmet edecek? Bu saatten sonra işe yarar bir çözüm haline gelebilir mi? Yoksa bir sonraki felakete hazırlık için kullanılabiliecek bir ürün çıkartabilir miyiz?

**Reply by Alone1206** [reply]
Merhaba, fikri sunan Kadir Beyle görüştüm. Kendisinin fikri aslında mantıklı. Geç kalındı evet ama bundan sonrası için de saha için ucuz bir çözüm olarak geliştirebilir. Hem doğrudan parçaların sahada tek bir laptop ile bile kurulup bir araya getirebilecek olması hem de aynı görevi yapan alternatiflerinden çok ucuz olarak üretilebilmesi ve açık kaynak olması neticesinde geliştirebilir olması açısından. Ben şahsen eğer bu projelendirilmek istenirse elimden geldiğince 3d baskı, devre, lehim, yazılım, cnc router desteği verebilirim.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebinizde yeterli detay olmadığı için Legal ekibin kontrolünden henüz geçemedi.

**Reply by Alone1206** [reply]
Merhaba Ramazan Bey, ne gibi detaylar lazım? Ana fikir sahibi hocamız github kullanıcısı değil ama kendisiyle ben iletişimdeyim.

**Reply by kdrysk** [reply]
ramazan bey detaylar konusunda yardımcı olmak isterim merak edilen konuları sorabilirsiniz

**Comment by mustafadilmac** [comment]
https://github.com/acikkaynak/deprem-yardim-projesi/discussions/64

Buradaki yerde fikrin benzeri için paylaşımda bulundum hala geç değil!

Daha basit hali ile ortak bir Wi-Fi listesi hızlıca toparlayabiliriz.

**Comment by ramazansubasi** [comment]
Arkadaşlar bu projenin bir çok çözülmesi gereken sorunu var, o yüzden mümkünse çok fazla efor harcamamak adına kapatılması daha iyi olur:

- Kameranın gece görüş olması gereliyor.
- Kameranın enrejisi
- Yıkılmış bir binada oluşacak katman nedeni ile wifi kalitesinin düşmesi
- Çevrede enerji olmaması nedeni ile modem vb cihazların çalıştırılamaması
- Kamerayı depremzedenin bulunduğu yere nasıl indireleceği sorunu? Beton blokların arasından kamerayı nasıl aşağı indireceksiniz.

**Reply by Alone1206** [reply]
Merhaba projede hiçbir şekilde modem gerekmiyor. Sadece cihazla telefon local ağda haberleşse yetiyor. Kamera sistemi 2 adet kalem pille veya küçük bir powerbankle bile çalışabilir herhangi bir yüksek güç tüketimi yok. Kamera çok ucuz fiyatlı olacağı için gerekirse sistem iple veya çubukla bile sallandırılabilir ki ana hedef zaten 1 metre civarında içeriyi görmesi. Bu sistemle aynı görevi yapan ama 600-700 lira bandında satılan cihazlar varken bu proje yaklaşık ünite başına 200 lira civarı tutuyor ve aynı görevi yapıyor. Artı olaraktan sahada bile tek bir laptopla güç olan bir yerde kurulup sonra enkazlara dağıtılabilir. Evet wifi kalitesi düşmesi konusunda haklısınız o da benim aklımda bir soru ama çok derine indirilmez veya relay görevi yapacak bir node cihazı daha konulabilir. Yaklaşık zaten sistem pille birlikte 100 gramı aşmayacaktır. gece görüşünden ziyade led takılabilir onboard güç çıkışına bağlı.  Zaten kendi içerisinde görüntü ile canlı olarak oynanabilecek bir arayüzü de içeriyor. Gerekirse görüntüde yüz de tanıyabiliyor.

**Reply by ramazansubasi** [reply]
Hocam yazdıklarında teknik olarak çok fazla hata var:


- İple veya çubukla nereye sallandırıyorsun? Kocaman bir beton blok var önünde. O beton blokları nasıl deleceksin?
- şuan bu sistem gerçek bir ihtiyacı çözse 600-700 liralık cihazlar neden kullanılmıyor mesela? 200 liraya mal edince ucuzladı diye kullanılacak mı sence?
- sahanın hiç bir yerinde elektrik yok, laptopun şarjı bitince ne yapacaksınız?
-  wifi kalitesinde dolayı derine inmeyeceksiniz neye yarayacak? Relay node'u o beton bloklarında arasında nereye ve nasıl konumlandıracaksın?

**Reply by Alone1206** [reply]
Kit halinde plug&play ve ithal oldukları için 600-700₺ bandında olduğunu düşünüyorum onun haricinde matah hiçbir yönleri yok. Çin malı adi bir cihaz.
Fikir sahibinin ana tasarım amacı anladığım kadarıyla çöküntülerdeki deliklerinden içerinin gözetlenmesi.
Sahadan kastım gördüğüm kadarıyla yardımların ulaştığı depo gibi bölgeler ve koordinasyon noktaları var belirli yerlerde oralarda da elektirk var. Sahadan kastım buydu. Ha zaten harekete geçilse bu proje için önden kurulup tam kit halinde biz de plug&play yaparız gelecekte olabilecek depremlerde. Laptop ile kurulum adımı ortadan kalkar sadece telefon yeterli duruma gelir.
(https://github.com/acikkaynak/deprem-yardim-projesi/discussions/31#discussioncomment-4943737) buraya attığım reply'da da dediğim gibi ana fikir sahibi bu deprem için düşünüyor projeyi ama ben gelecek için geliştirilmesi taraftarıyım. Keza şu anda espcamleri bulsak en erken pazartesi alana varırlar. Çok çok geç olur diye düşünmekteyim. Bu sebeple kapatalım demenize karşı çıkıyorum. Çünkü yeterli bir ekiple çin malı muadilinden daha ucuz ve gelişmişini yapabiliriz.

**Reply by ramazansubasi** [reply]
Hocam kritik burada: yeterli bir ekiple çin malı muadilinden daha ucuz ve gelişmişini yapabiliriz.

Ucuz olarak geliştirdin diyelim. Sahada kullanılacak mı? Pahalı olanlar niye sahada yok şuan?  Pahalı oldukları için mi yoksa saha şartlarında fayda sağlama oranın çok az olacağı için mi?

**Reply by kdrysk** [reply]
ramazan bey nekadar hızlı kestirip projeyi kapatabiliyorsunuz ve ne hakla bunu söyleme yetkinliğini kendinizde görüyorsunuz burada kendi yorumunuzu yapabilirsiniz yanlız bu başlığı kapatma kararını sizin verebileceğinizi düşünmüyorum. aynı zamanda bu projeyi derinleri kontrol etmek için değil sadece deprem alanındaki içeri girmesi tehlikeli bölgelerin 1 veya 2 metrelik mesafelerden yada küçük açılan deliklerden içerisinin kontrol edilmesini amaçladım.Sadece insanlara yardım etmek için bu projeyi ortaya koydum benim ismim burda varmış yokmuş hiç mühim değil bu saaten sonra bu projenin başlanıp gönderilmesi geç ama bundan sonra bu projeyi yapacak olan gençlere desteğim tamdır bana ulaşırlarsa yardımcı olacağım ...Bu arada merak etmeyin esp32'nin hotspot'u 2 metrelik veri aktarımına gayet müsait dökümantasyon taramanızı ve iş hayatınız için okuyup tecrübe edinmenizi tavsiye ederim.Aynı zamanda burada benzer çin muadillerinin kullanılmaması esp32 nin kullanılamayacağı anlamına gelmez.İyi çalışmalar dilerim.(Not:esp32 hotspot üzerinden  bataryaya bağlanarak html ile basit bir arayüz üzerinden telefona görüntü aktarımı tecrübe edilmiştir bilginize sunulur.)

**Reply by Alone1206** [reply]
Ramazan bey projeyi gerçekleştirdim. 4 metreden stabil bağlantı alınıyor 3 duvar arakasından.
